websocketOutportDriver/sender/webSocketSender.go

```diff
@@ -79,7 +79,7 @@ func (w *webSocketSender) AddClient(wss WSConn, remoteAddr string) {
 
 	w.clientsHolder.AddClient(client)
 
-	w.acknowledges.AddEntryForClient(remoteAddr)
+	w.acknowledges.AddEntryForAddress(remoteAddr)
```

Comment:
Kindly reminder for +1 for the comment below from @iulianpascalau 
You should not call `w.acknowledges.AddEntryForAddress(remoteAddr)` if `w.withAcnkowledge == false`

Reply:
done :medal_military: